Doc/library/importlib.rst

```diff
@@ -985,6 +985,17 @@ find and load modules.
       :exc:`ImportError` is raised.
 
 
+.. class:: NamespacePath(name, path, path_finder)
+
+   Represents a namespace package's path.
+
+   It uses the module *name* to find its parent module, and from there it looks
+   up the parent's :attr:`module.__path__`.  When this changes, the module's own
+   path is recomputed, using *path_finder*. The initial value is set to *path*.
+
+   For top-level modules, the parent module's path is :data:`sys.path`.
+
+
```

Comment:
Can we use `.. versionadded:: next` + what's new entry? While this only exposes a private class, I think users should be notified more than just with a blurb.